Makefile

```diff
@@ -28,7 +28,19 @@ everything-curl.html: uni.md $(MDS)
 	rm -rf everything-curl
 	mkdir -p everything-curl
 	cp -p `grep -oe 'img src="[0-9a-z/.-]*' everything-curl.html | cut -c10-` everything-curl/
-	cp everything-curl.html everything-curl/index.html
+	>> everything-curl/index.html echo '<!doctype html>'
+	>> everything-curl/index.html echo '<html lang="en">'
+	>> everything-curl/index.html echo '  <head>'
+	>> everything-curl/index.html echo '	<meta charset="utf-8">'
+	>> everything-curl/index.html echo '	<meta name="viewport" content="width=device-width, initial-scale=1">'
+	>> everything-curl/index.html echo '	<link rel="icon" href="data:;base64,iVBORw0KGgo=">'
```

Comment:
This is to disable favicon requests by browser. See https://stackoverflow.com/a/13416784